### DISCUSSION TITLE
problem p-table & virtualscroll / PrimeNG v.15 lts

### DISCUSSION THREAD

**Discussion by CesarVillajo** [primefaces/community#1142] [PrimeNG]
Hello!! I have a great problem in p-table with virtualScroll.

In the header it has 2 <tr> and 45 columns
In the second <tr> I have many select, multiselect, input and input elements.

When I select an option from any of these items, virtualScroll resets the scroll bar to vertical and horizontal positions.

My problem is that if I am in column 25 and select any option, the horizontal scroll moves to the beginning, losing sight of the column in which I was selecting the option.
How do I keep this from happening?

Thanks
I have PrimeNG v.15 liters.
If I do not use virtualScroll and use a pager, this does not happen.

<p-table  #tt
  [columns]="cols"  [value]="testigosRU"   dataKey="id_testigo"   sortField="id_testigo"  id="tablaRUS"   exportFilename="testigos"
  [(first)]="first"   [(selection)]="testigosRUSeleccionados"   selectionMode="multiple"   (onRowSelect)="seleccionarEvento($event)"
  (onRowUnselect)="deseleccionarEvento($event)"  (onHeaderCheckboxToggle)="seleccionarTodoEvento($event)"
  (onFilter)="filtradoEvento($event)"  (onColReorder)="onColReorderCustom($event)"

  [scrollable]="true"
  scrollHeight="477px"  
  [virtualScroll]="true"
  [virtualRowHeight]="36"  
  [rows]="25"
 expandableRows="true"
  [reorderableColumns]="true"
  [resizableColumns]="true"
  columnResizeMode="expand"
  styleClass="p-datatable-gridlines"
>

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue precisely?

**Comment by CesarVillajo** [comment]
Sorry i cant share this proyect.
I have solved the problem by downloading version 12 only for the table.
Although now the header jumps, the horizontal scroll does not restart every time I make a selection in a multiselect or in an input.
In any case, I tell the Prime developers to try having a table with 40 columns and multiselect in the headers to see what happens.
I don't think they have tested anything similar.

**Reply by cetincakiroglu** [reply]
You don't need to share the project, you can recreate the issue in the simplest way possible. If not we cannot identify the issue, thus, we cannot investigate if there is a bug.

**Comment by cetincakiroglu** [comment]
Closing due to the inactivity.
